src/hammer-tech/hammer_tech.py

```diff
@@ -672,7 +672,10 @@ def prepend_dir_path(self, path: str, lib: Optional[Library] = None) -> str:
             if len(matching_installs) == 1:
                 install = matching_installs[0]
                 if install.base_var == "":
-                    base = self.path
+                    if "cache" in install.path.lower():
```

Comment:
In addition to what Colin said, perhaps instead of matching any lowercase "cache", maybe restrict this to a specific install.path?
```suggestion
                    if install.path == "tech_cache":
```
or something

Reply:
Yeah, I think I agree with you two here. I still think that having the path be "tech_cache" is a bit unsatisfactory because it has to be such a specific string, but I can't think of a different way of doing it...